### ISSUE TITLE
unexpected token '}'

### ISSUE THREAD

**Issue by Kiyozz** [evanw/esbuild#962]
I’m using esbuild 0.9.1

My config

```js
{
    entryPoints: files, // all my sources
    outdir: 'dist',
    platform: 'node',
    format: 'cjs',
    target: 'node10',
}
```

I’d like to mention, I'm not using `bundle: true`

## Problem

Any export default statement failed at runtime with `unexpected token '}'` error.

Output from esbuild 

```js
0 && (module.exports = {
  default
})
```

It seems that default is treated as keyword `default`

Tested with node 10/12/14/15

I tested both minification or normal

Screenshot

<img width="356" alt="CleanShot 2021-03-12 at 18 49 10@2x" src="https://user-images.githubusercontent.com/12450333/110978415-a4834780-8363-11eb-8360-27fd3e1ebe59.png">

**Comment by evanw** [comment]
Really sorry about that. Will fix this soon. Closing as a duplicate of #960.

**Comment by evanw** [comment]
This has been fixed in version 0.9.2.
